### PR TITLE
Format phone number

### PR DESCRIPTION
**Pull request by SabrinaSnider**
Formats phone numbers on organization cards to a +#(###) ### - ### format.

image

### CODE REVIEW

File: public/js/discover/searchList.js [hunk 1]
```diff
@@ -71,3 +76,29 @@ export function selectCard(id = null, scroll = false) {
...
+  } else if (number.length == 10) {
+    return (
+      '+' +
+      number.substring(0, 1) +
+      ' (' +
+      number.substring(1, 4) +
+      ') ' +
+      number.substring(4, 7) +
+      ' - ' +
+      number.substring(7, 10)
```
Comment (malcolmrjones) [inline public/js/discover/searchList.js:99]: Assign a variable to each of the different substrings of the phonenumber then use the variables when constructing the string. 

**Parts of a Phone Number**
image

Comment (malcolmrjones) [inline public/js/discover/searchList.js:99]: nit: If you want this is a good place for using JS String interpolation.
See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

File: public/js/discover/searchList.js [hunk 2]
```diff
@@ -71,3 +76,29 @@ export function selectCard(id = null, scroll = false) {
...
+
+/**
+ * Formats a raw phone number into a +# (###) ### - #### format.
+ * @param {string} number The phone number unformatted.
+ * @return {string} The formatted phone number.
+ */
+function formatPhoneNumber(number) {
+  if (number.length == 9) {
+    return (
+      '(' + number.substring(0, 3) + ') ' + number.substring(3, 6) + ' - ' + number.substring(0, 3)
```
Comment (malcolmrjones) [inline public/js/discover/searchList.js:88]: Same comments here as for else if statement.

File: public/js/discover/searchList.js [hunk 3]
```diff
@@ -71,3 +76,25 @@ export function selectCard(id = null, scroll = false) {
...
+    const exchangeCode = number.substring(3, 6);
+    const lineNumber = number.substring(6, 9);
+    return `(${areaCode})-${exchangeCode}-${lineNumber}`;
+  } else if (number.length == 10) {
+    const countryCode = number.substring(0, 1);
+    const areaCode = number.substring(1, 4);
+    const exchangeCode = number.substring(4, 7);
+    const lineNumber = number.substring(7, 10);
+    return `+${countryCode} (${areaCode})-${exchangeCode}-${lineNumber}`;
+  } else {
```
Comment (brucezheng) [inline public/js/discover/searchList.js:97]: Could use a bit of recursion to save code here: ```return `+${countryCode} ${formatPhoneNumber(number.substring(1, 10))}````.

**Pull request merged**